### ISSUE TITLE
Cannot drush ssh or ssh into PR servers

### ISSUE THREAD

**Issue by ElijahLynn** [NBCUOTS/flo#178]
As a developer I need SSH access to our servers to properly debug.

**Comment by bleen** [comment]
Can you make your specific case?

**Comment by ElijahLynn** [comment]
See if files exist, check file and folder permissions.

**Comment by ElijahLynn** [comment]
Also, drush ssh to be able to run mysql console every now and then.

In addition drush ssh to do simple vget and vset, enable/disable modules. The normal dev workflow just doesn't work otherwise.

**Comment by ericduran** [comment]
This has always been on purpose. If there's problems we should fix it but we shouldn't open permission etc..

To be fair we can now use @conortm https://github.com/conortm/ghkeys to allow anyone into the server, I just no it'll be a slippery slope

**Comment by ericduran** [comment]
If we get Publisher7 it's own jenkins secondary server like we have on the brands, I'll be more open to this, but right now that would be given access to our entire Flo infra :-1:

**Comment by cweagans** [comment]
Having access to the entire Flo infra really wouldn't be that bad of a thing, IMO. Nobody on the P7 team is going to do anything nefarious, and opening it up would allow for faster resolution of problems that would otherwise require you (@ericduran) to go look at things. You're a busy guy. Things like this seem like things the team should be able to do. Just my $0.02.

**Comment by ElijahLynn** [comment]
Yeah, I don't see any issues with that either. We trust people with push access to master this isn't much different.

**Comment by ElijahLynn** [comment]
I read an article a while back from Etsy or one of those companies that they have the dev team own the entire CI process/infrastructure as it makes everyone more involved and they can help improve it.

**Comment by ElijahLynn** [comment]
cc @joannalisowski
